15_totalIntegers/solution/totalIntegers-solution.js

```diff
@@ -0,0 +1,24 @@
+// We could define this helper function inside of totalIntegers however that would not be best practice as everytime that the recursive totalIntegers is run it would re-create the isObject function when it could just re-use it.
```

Comment:
Not sure we need this comment. I would think a quick reminder that `typeof null` is `'object'` would be sensible (it *always* catches me out).

Reply:
Yeah, I agree that this comment should maybe just focus on `typeof null` behavior